pkg/util/test/util.go

```diff
@@ -217,3 +218,8 @@ func ActivateCRDWhenCreated(fakeClient *fakeextclient.Clientset, crdName string)
 		return false, nil
 	})
 }
+
+// ValidateStorageCluster validates a StorageCluster spec
+func ValidateStorageCluster(cluster *corev1alpha1.StorageCluster) error {
+	return fmt.Errorf("not implemented yet")
```

Comment:
You can move the implementation here from the common_test file.
Also you will have to move - 
UninstallStorageCluster() and ValidateUninstallStorageCluster(), so we can use it in spawn.